src/react/hoisting.js

```diff
@@ -115,6 +115,7 @@ function canHoistAbstract(realm: Realm, abstract: AbstractValue, residualHeapVis
     for (let scope of scopes) {
       const currentAdditionalFunction = residualHeapVisitor.commonScope;
       invariant(currentAdditionalFunction instanceof FunctionValue);
+      // $FlowFixMe: there is no such property
       if (scope === currentAdditionalFunction.parent) {
```

Comment:
So this is likely a bug? Maybe leave behind a TODO?

Reply:
I removed the `.parent` property from FunctionValues a while ago as it wasn't well defined. @trueadm, what's the intention behind the `.parent` check? Doesn't seem to affect anything.